src/extract/src/html/extract-regexps-html.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Description: This file takes in a html file from extract-regexes.pl, finds all the script 
+# tags, and combine the JS in them into a temporary js file. It then sends the path of the
+# temporary js file back to extract-regexes.pl to let it pipeline the js file to the javascript
+# extractor. After extract-regexes.pl finishes extracting, The temporary JS file will be 
+# deleted by extract-regexes.pl. 
+
+from bs4 import BeautifulSoup
+import sys
+import subprocess
+import json
+
+file_path = sys.argv[1]
+with open(file_path) as fp:
+    soup = BeautifulSoup(fp, 'html.parser')
+
+js_from_html = ''
```

Comment:
Is it appropriate to merge all of the JS script tags like this, or would it be better to make one file per <script>? I am not sure the DOM JS semantics, could we get compilation failures as a result?